tests/cli/main.cpp

```diff
@@ -388,3 +388,30 @@ BOOST_AUTO_TEST_CASE( cli_set_voting_proxy )
    }
    app1->shutdown();
 }
+
+///////////////
```

Comment:
test cases for exit command dont need a test case, it is actually not testing anything here.

Reply:
in test case I check next command quit:

con.wallet_api_ptr->quit();

Reply:
just executing the command does not mean it is testing anything.

Reply:
what should be done to correctly testing quit command ? can you direct me in right way please ?

Reply:
just remove the test case fully, not needed.

Reply:
ok understood ! Thanks ! Just thought test case needed to check if there are any exceptions while quiting wallet

Reply:
removed fully test case